_api-reference/cluster-api/cluster-voting-configuration.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Cluster voting configuration
+nav_order: 55
+parent: Cluster APIs
+---
+
+# Cluster voting configuration
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Cluster voting configuration API updates and clears voting configurations.
+
+# Endpoints
+
+The `POST` method updates the cluster voting configuration by excluding certain node IDs or names.
+
+<!-- spec_insert_start
+api: cluster.post_voting_config_exclusions
+component: endpoints
+omit_header: true
+-->
+```json
+POST /_cluster/voting_config_exclusions
+```
+<!-- spec_insert_end -->
+
+The `DELETE` method clears any cluster voting configuration exclusions.
+
+<!-- spec_insert_start
+api: cluster.delete_voting_config_exclusions
+component: endpoints
+omit_header: true
+-->
+```json
+DELETE /_cluster/voting_config_exclusions
+```
+<!-- spec_insert_end -->
+
+
+<!-- spec_insert_start
+api: cluster.post_voting_config_exclusions
+component: query_parameters
+-->
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
```

Comment:
`node_ids` or `node_names` is required for the POST method.  Please update the spec.

Reply:
We can't update the spec to require both of them, since this would change the behavior of the API. I'll update the descriptions to say that one or the other is required.